README.md

```diff
@@ -1,323 +1,299 @@
+```
+For example, if you want to run the `discv4` test, here is how the command would look:
 
-This section is a quickstart of command line options covering typical Hive usage scenarios.
+```bash
+./hive --sim devp2p/discv4 --client go-ethereum_latest --loglevel 6
+```
 
-For additional Windows-specific command line options please check the section following.
+## Quickstart command lines
 
-## Consensus test
-To execute the consensus tests on parity run the latest simulator:
+This section is a quick start guide for command line options covering typical hive usage scenarios.
 
+### Consensus test
+The following command will run consensus tests on the parity client
 ```text
-   --docker-noshell
+   --sim consensus
    --client parity_latest
-   --sim ethereum/consensus2
-   --sim.rootcontext
-   --results-root /mytests/test
+   --results-root /mytests/tests
 ```
 
-## Devp2p tests
+### Devp2p tests
 
-These run devp2p tests, mainly Discovery tests.
+The following command will run devp2p tests on geth: 
 
 ```text
     --sim devp2p
-    --sim.rootcontext
-    --test none
-    --loglevel 6
-    --docker-noshell
-    --results-root /mytests/test
     --client go-ethereum_latest
-    --sim-parallelism 1
+    --loglevel 6
+    --results-root /mytests/tests
+    --sim.parallelism 1
 ```
+The `--sim.parallelism` flag will set the maximum number of clients against which to run the simulation in parallel. 
 
-## Sync simulation
+### Sync simulation
 
-These run a test verifying that a blockchain can be synced between differing implementations.
+The following command will run a test verifying that a blockchain can be synced between differing implementations (in this case, parity and geth):
 
-      --sim ethereum\\\\sync
-      --sim.rootcontext
-      --test none
-      --loglevel 6
-      --docker-noshell
-      --results-root /mytests/test
+      --sim sync
       --client go-ethereum_latest,parity_latest
-      --sim-parallelism 1
+      --loglevel 6
+      --results-root /mytests/tests
 
-## Generate a blockchain
+### Iterating on bugfixes locally
 
-This uses Hive to generate a blockchain based on a genesis file for subsequent testing in sync simulations.
+If you are testing locally and want to make changes to the simulation or client, run hive with the flag `--docker-nocache <simulation or client name>` so that hive rebuilds the container from scratch.
 
-      --loglevel 6
-      --chainGenerate
-      --chainLength 10
-      --chainOutputPath C:\\Ethereum\\OutputPath
-      --chainGenesis C:\\Ethereum\\Input\\Genesis.json
-      --chainBlockTime 30
+If you want to rebuild both, separate the names with a `,` as such: 
+```text
+--docker-nocache devp2p,go-ethereum_latest
+```
 
-## Iterating on bugfixes locally
+# Adding a Simulation
+There are two components to a simulation: 
+1. a **simulation program written in Go** using the `hivesim` test API to coordinate the execution of your desired test; and
```

Comment:
I would strongly prefer that we don't say `hivesim` is required. From hive's point of view, a simulation is any program using the simulation API. It doesn't need to be written in Go, and doesn't need to use package hivesim. For documenting package hivesim, we should add the tutorial content you wrote here to the GoDoc instead, i.e. it should be in hivesim/doc.go. Then you can just link to that from here.